### PR TITLE
feature(charts): hermes chart fixes, bech32 updates, ibc bridge test

### PR DESCRIPTION
**Pull request by steezeburger**
## Summary
Fixes hermes helm chart

## Background
We want to run hermes via helm chart so we can automate e2e testing.

## Changes
* add 2nd funded celestia account
* use this account's key in hermes
* fix hermes chart values
* add new chart `bridge-test`
* add new justfile `dev/bridgetester.just`
* add `AstriaSequencerHrpPrefix` to evm rollup geth-genesis

## Testing
- `just deploy-cluster`
- `just deploy-ibc-test-infra`
  - deploys ingress controller
  - deploys celestia-local
  - deploys single astria sequencer
    - `./charts/sequencer/values.yaml`
    - `./dev/values/validators/single.yaml`
  - deploys evm rollup
    - `./charts/evm-rollup/values.yaml`
    - `./dev/values/rollup/dev.yaml`
    - `./dev/values/rollup/ibc-bridge-test.yaml`
  - deploy hermes
    - `./charts/hermes/values.yaml`
    - `./dev/values/hermes/local.yml`
- `just deploy-bridge-tester`
  - initContainer handles initializing bridge account on the sequencer
  - container runs scripts
    - `celestia-appd keys add`
    - `celestia-appd tx ibc-transfer transfer`
    - checks balance before and after and fails if ibc-transfer did not go through

## Metrics
- List out metrics added by PR, delete section if none. 

## Breaking Changelist
- Bulleted list of breaking changes, any notes on migration. Delete section if none.

## Related Issues
Link any issues that are related, prefer full github links.

closes

### CODE REVIEW

File: dev/values/hermes/local.yml [hunk 1]
```diff
@@ -27,38 +34,41 @@ chains:
       interval: 1s
     gasPrice: 1
     gasDenom: nria
-    trustThreshold: 2/3
+    trustThreshold: 1/3
```
Comment (joroshiba) [inline dev/values/hermes/local.yml:37]: this should probably be 2/3 to match tendermint?

File: dev/values/hermes/local.yml [hunk 2]
```diff
@@ -27,38 +34,41 @@ chains:
...
+    gasPrice: 0.0026
+    gasMultiplier: 1.2
+    defaultGas: "1000000"
+    maxGas: "10000000"
     maxMsgNum: 30
-    maxTxSize: "180000"
+    maxTxSize: "2097152"
     clockDrift: 5s
     maxBlockTime: 30s
     trustThreshold: 1/3
```
Comment (joroshiba) [inline dev/values/hermes/local.yml:74]: Also should probably be 2/3

File: dev/values/hermes/local.yml
Comment (joroshiba) [inline dev/values/hermes/local.yml:1]: we should probably add a trustingPeriod of '14days' for both chains

File: dev/values/hermes/local.yml (hunk 1 above)
Comment (steezeburger) [reply dev/values/hermes/local.yml]: Updated!
I got these values from https://github.com/astriaorg/hermes/blob/astria/config-astria-celestia.toml
Should I update there as well? Not 100% sure of the implication/meaning of trustThreshold

Comment (joroshiba) [reply dev/values/hermes/local.yml]: > The trust threshold defines what fraction of the total voting power of a known and trusted validator set is sufficient for a commit to be accepted going forward. - [hermes docs](https://hermes.informal.systems/documentation/commands/path-setup/clients.html?highlight=trust%20threshold#create-client)

Basically 2/3 is the threshold for CometBFT to accept a block so I don't think it makes sense to accept lower than that.

**Review by joroshiba — APPROVED** [review]
looks good enough to me, we can merge and then clean up

**Pull request merged**